docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +77,64 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+RabbitMQ Installation
+.....................
+
+If in ``verdi status`` RabbitMQ is not connected, first check that RabbitMQ is actually running:
```

Comment:
It should be noted here that none of this applies to Windows (WSL), where we recommend installing the Windows version of RabbitMQ: https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/installation.html#windows-subsystem-for-linux-ubuntu

Reply:
Yeh well my suggestion for windows would be; don't use it because its dumb 😜. I'll maybe change then name to `RabbitMQ Installation (Unix)` then? Do you have anything specific tips/links for WSL?

Reply:
Nah, haven't had any problems with it 🤣